addons/purchase_requisition/static/src/xml/purchase_order_line_compare.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+
+<t t-extend="ListView.buttons" t-name="PurchaseOrderLineCompare.Buttons">
+    <t t-jquery="button.o_list_button_discard" t-operation="after">
+        <button class="btn btn-secondary o_clear_qty_buttons mr-2" type="button" style="display:none;">
```

Comment:
Not sure to understand why there is a `display: none` here ?
If it's really needed, can you check if the `d-none` Bootstrap class  can be used instead of defining a style directly in the tag ?